frontend/view-edit-events/view-event.component.tsx

```diff
@@ -121,6 +121,7 @@ export type SingleEvent = {
   totalConvertedToClients?: number;
   leadGenders?: any;
   clientGenders?: any;
+  leads?: any;
```

Comment:
Feels like cheating a bit, but it was being done on `leadGenders` and `clientGenders`

Reply:
fair enough